lib/v1.0/python/pprzlink/udp.py

```diff
@@ -0,0 +1,153 @@
+"""
+PPRZLINK UDP to Python interface
+"""
+from __future__ import absolute_import, division, print_function
+
+import threading
+import socket
+
+# used to run the test from this directory
+# any better implementation ?
+import os
+import sys
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
```

Comment:
That is what I did in the beginning, but then if I try to run the test script with `python serial.py` for instance, I get:
```
Traceback (most recent call last):
  File "serial.py", line 4, in <module>
    import serial
  File "/home/gautier/dev/pprzlink/lib/v1.0/python/pprzlink/serial.py", line 6, in <module>
    from .message import PprzMessage
ValueError: Attempted relative import in non-package
```
(One of) the workaround I found on Internet is the one I used in `udp.py`. But maybe I missed something.